bpf/Makefile

```diff
@@ -95,9 +95,10 @@ bpf_overlay.o: bpf_overlay.ll
 
 NETDEV_OPTIONS = $(LB_OPTIONS) \
 	-DENABLE_IPV4:-DENABLE_IPV6:-DLB_L3:-DLB_L4:-DENABLE_HOST_SERVICES_UDP:-DENABLE_HOST_SERVICES_TCP:-DENABLE_NODEPORT:-DENABLE_EXTERNAL_IP:-DENABLE_MASQUERADE \
-	-DENABLE_IPV4:-DENABLE_IPV6:-DLB_L3:-DLB_L4:-DENABLE_HOST_SERVICES_UDP:-DENABLE_HOST_SERVICES_TCP:-DENABLE_NODEPORT:-DENABLE_EXTERNAL_IP:-DENABLE_DSR
+	-DENABLE_IPV4:-DENABLE_IPV6:-DLB_L3:-DLB_L4:-DENABLE_HOST_SERVICES_UDP:-DENABLE_HOST_SERVICES_TCP:-DENABLE_NODEPORT:-DENABLE_EXTERNAL_IP:-DENABLE_DSR \
+	-DENABLE_IPV4:-DENABLE_IPV6:-DLB_L3:-DLB_L4:-DENABLE_HOST_SERVICES_UDP:-DENABLE_HOST_SERVICES_TCP:-DENABLE_EXTERNAL_IP:-DENABLE_DSR:-DFROM_HOST
 
-MAX_NETDEV_OPTIONS = $(MAX_LB_OPTIONS) -DENABLE_DSR
+MAX_NETDEV_OPTIONS = $(MAX_LB_OPTIONS) -DENABLE_DSR -DENABLE_MASQUERADE
```

Comment:
If you add `ENABLE_DSR_HYBRID` to the list, this should improve the coverage even more.

Also, you could add `ENABLE_HOSTPORT` which was recently introduced in #10592.

Reply:
Thanks! Done.